content/docs/getting-started/installing.md

```diff
@@ -9,9 +9,10 @@ Scratch Addons is available in these stores.
-| Chrome Web Store | [![Install for Chrome Web Store](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=install&color=4285F4)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco) | Google Chrome 80+<br />Microsoft Edge 80+<br />Opera 67+<br />Brave 1.3+<br />Vivaldi 2.11+<br />*Chromium 80+* | Windows 7+<br />OS X / MacOS 10.11+<br />Chromebooks less than ~6 years old
-| Add-ons for Firefox | [![Install for Add-ons for Firefox](https://img.shields.io/amo/v/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=install&color=FF7139)](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/) | Mozilla Firefox 86+ | Windows 7+<br />OS X / MacOS 10.12+
-| Microsoft Edge Add-ons | [![Install for Microsoft Edge Addons](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=microsoftedge&logoColor=white&label=install&color=0078D7&prefix=v&query=%24.version&url=https%3A%2F%2Fmicrosoftedge.microsoft.com%2Faddons%2Fgetproductdetailsbycrxid%2Filiepgjnemckemgnledoipfiilhajdjj)](https://microsoftedge.microsoft.com/addons/detail/iliepgjnemckemgnledoipfiilhajdjj) | Microsoft Edge 80+ | Windows 7+<br />OS X / MacOS 10.11+
+| Chrome Web Store | [![Install for Chrome Web Store](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=cws&color=4285F4)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco)<br /><br />[Install](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco)| Google Chrome 96+<br />Microsoft Edge 96+<br />Opera 82+<br />Brave 1.33+<br />Vivaldi 5.0+<br />*Chromium 96+* | Windows 7+<br />OS X / MacOS 10.11+<br />Chromebooks less than ~6 years old
+| Add-ons for Firefox | [![Install for Add-ons for Firefox](https://img.shields.io/amo/v/scratch-messaging-extension?style=flat-square&logo=firefox-browser&logoColor=white&label=moz&color=FF7139)](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/)<br />[Install](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/) | Mozilla Firefox 109+ | Windows 7+<br />OS X / MacOS 10.12+ <!-- Android 5.0+ -->
+| Microsoft Edge Add-ons | [![Install for Microsoft Edge Addons](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=microsoftedge&logoColor=white&label=mse&color=0078D7&prefix=v&query=%24.version&url=https%3A%2F%2Fmicrosoftedge.microsoft.com%2Faddons%2Fgetproductdetailsbycrxid%2Filiepgjnemckemgnledoipfiilhajdjj)](https://microsoftedge.microsoft.com/addons/detail/iliepgjnemckemgnledoipfiilhajdjj)<br />[Install](https://microsoftedge.microsoft.com/addons/detail/iliepgjnemckemgnledoipfiilhajdjj)  | Microsoft Edge 96+ | Windows 7+<br />OS X / MacOS 10.11+
+| From source *(local development)* | See below | Chromium 121+<br />Mozilla Firefox 121+ | Windows 10+<br />MacOS 10.15+

```

Comment:
Remember that this section is for installing through extension stores. Should you want to mention about this, mention it on the next section ("From source"), and it doesn't have to be a table.

Reply:
Ideally I would mention it in both places.

The install page in the docs is a page people visit often and people rarely read it fully. I think it's fine to sacrifice consistency and structure for better usability.

Reply:
Pardon?

We are talking about the part that's about local development. It is unnecessary to put it on a section that's called "From stores", or else it would be confusing. You only need to move down there, so much for *usability* for you; moving it would be negligble for such usability!

Reply:
Sure. And what do you think about having actual "Install" links that look clickable?

Reply:
The buttons are adequate since (a) it is on the install column and (b) the badge/shield (that's what the image is called) has the word "install" and the version, which should mean "use this badge/shield (by clicking it) to install this version!"